amaznot/app/Http/Controllers/OrderController.php

```diff
@@ -112,4 +115,29 @@ private function getOrderItems(Request $request, $orderId)
 
         return $orderItems;
     }
+    
+    public function orderDetails(Request $request, $id)
+    {
+        //Check credentials
+        if ($redirect = parent::redirectOnNotUser($request))
+        {
+            return $redirect;
+        }
+
+        $orderDetails = OrderItem::with('order_items')->where('id', $id)->get();
+        $ordersDetails = OrderItem::paginate(10);
+
+        return view('pages.ordersitems', [
+            'orders' => $orderDetails,
+            'clearCart' => $request->query('clearCart')
+        ]);
+    }
+
+    // Delete an Order
+    public function deleteOrder(Request $request, $id)
+    {
+        $deleted = OrderItem::with('order_items')->where('id', $id)->delete();
```

Comment:
Can you make a test button in the orders page to make sure this works correctly? Also, user verification needs to be done for this method.